ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/enable_trunk_groups.schema.yml

```diff
@@ -0,0 +1,17 @@
+# yaml-language-server: $schema=../../../../plugins/plugin_utils/schema/avd_meta_schema.json
+# Line above is used by RedHat's YAML Schema vscode extension
+# Use Ctrl + Space to get suggestions for every field. Autocomplete will pop up after typing 2 letters.
+type: dict
+keys:
+  enable_trunk_groups:
+    documentation_options:
+      filename: Fabric Variables
+      table: VLAN Settings
+    type: bool
+    description: |
+      Enable Trunk Group support across eos_designs
+      Warning: Because of the nature of the EOS Trunk Group feature, enabling this is "all or nothing".
+      *All* vlans and *all* trunks towards connected endpoints must be using trunk groups as well.
+      If trunk groups are not assigned to a trunk, no vlans will be enabled on that trunk.
+      See "Details on enable_trunk_groups" below before enabling this feature.
```

Comment:
Same here - I think we are still not sure what will be done with documentation so we may need to revisit this statement (regarding the doc being below) later (maybe let's add some TODO or track somewhere?)

Reply:
All descriptions will be reviewed in the next rounds of PRs where we focus on building the documentation.